### ISSUE TITLE
Add `NCAR/cesm-lens-aws` repo to xdevbot watchlist

### ISSUE THREAD

**Issue by andersy005** [NCAR/xdev#116]
/add-repo campaign:analysis repo:NCAR/cesm-lens-aws﻿

**Comment by kmpaul** [comment]
This errored!  There was no webhook installed.  Can you check if the webhook is there or not?

**Comment by kmpaul** [comment]
Also, have you noticed that the link above to the `NCAR/cesm-lens-aws` repo has some extraneous characters in it that break the link?  The link actually reads as `https://github.com/NCAR/cesm-lens-aws%EF%BB%BF`.

**Comment by andersy005** [comment]
> Also, have you noticed that the link above to the `NCAR/cesm-lens-aws` repo has some extraneous characters in it that break the link? The link actually reads as `https://github.com/NCAR/cesm-lens-aws%EF%BB%BF`.

This is now fixed. Also, the Github action wasn't the culprit. For reason unbeknownst to me, there was some weird character at the end of the text, and this character wasn't being rendered. It's when I pasted the text body into an editor that I found it:

![Screen Shot 2020-10-13 at 3 55 21 PM](https://user-images.githubusercontent.com/13301940/95934580-71c22c80-0d8e-11eb-82fc-7b2cd5a51eb1.png)
